packages/react-sdk/src/state/useWhiteboardManager.test.tsx

```diff
@@ -50,10 +50,12 @@ describe('useWhiteboardManager', () => {
   });
 
   it('hook should throw without context', () => {
-    const { result } = renderHook(() => useWhiteboardManager());
+    jest.spyOn(console, 'error').mockImplementation(() => {});
```

Comment:
Shouldn't there be a `mockRestore` after each test?

(for all mocked console.*)

Reply:
of course there should be! 👯‍♂️👯‍♂️ I've updated all occurrences...